File: c++/spring2024/EyeOfSauron/EyeOfSauron.cpp
/// \file EyeOfSauron.cpp
/// \author Chad Hogg
/// My solution to https://open.kattis.com/problems/eyeofsauron

#include <iostream>
#include <string>

int main()
{
    std::string text;
    std::cin >> text;
    std::size_t paren = text.find ('(');
    if (paren == text.size () - paren - 2) {
        std::cout << "correct\n";
    }
    else {
        std::cout << "fix\n";
    }
    return 0;
}

File: python/fall2024/EyeOfSauron/EyeOfSauron.py
# My solution to https://open.kattis.com/problems/eyeofsauron from August 29, 2024

line = input()
if len(line) % 2 == 1:
    print('fix')
elif line[len(line) // 2 - 1] != '(' or line[len(line) // 2] != ')':
    print('fix')
else:
    print('correct')